Detectors/Raw/src/RawFileReader.cxx

```diff
@@ -56,18 +56,18 @@ std::string RawFileReader::LinkData::describe() const
 //____________________________________________
 void RawFileReader::LinkData::print(bool verbose, const std::string& pref) const
 {
-  LOGF(INFO, "%s %s FEE:0x%04x CRU:%4d Lnk:%3d EP:%d RDHv%d Src:%s | SPages:%4d Pages:%6d TFs:%6d with %6d HBF in %4d blocks (%d err)",
-       pref, describe(), int(RDHUtils::getFEEID(rdhl)), int(RDHUtils::getCRUID(rdhl)), int(RDHUtils::getLinkID(rdhl)),
-       int(RDHUtils::getEndPointID(rdhl)), int(RDHUtils::getVersion(rdhl)),
-       RDHUtils::getVersion(rdhl) > 5 ? o2h::DAQID::DAQtoO2(RDHUtils::getSourceID(rdhl)).str : "N/A",
-       nSPages, nCRUPages, nTimeFrames, nHBFrames, int(blocks.size()), nErrors);
-  if (verbose) {
-    for (int i = 0; i < int(blocks.size()); i++) {
-      std::stringstream counts;
-      counts << '#' << std::setw(5) << i << " | ";
-      blocks[i].print(counts.str());
-    }
-  }
+//  LOGF(INFO, "%s %s FEE:0x%04x CRU:%4d Lnk:%3d EP:%d RDHv%d Src:%s | SPages:%4d Pages:%6d TFs:%6d with %6d HBF in %4d blocks (%d err)",
+//       pref, describe(), int(RDHUtils::getFEEID(rdhl)), int(RDHUtils::getCRUID(rdhl)), int(RDHUtils::getLinkID(rdhl)),
+//       int(RDHUtils::getEndPointID(rdhl)), int(RDHUtils::getVersion(rdhl)),
+//       RDHUtils::getVersion(rdhl) > 5 ? o2h::DAQID::DAQtoO2(RDHUtils::getSourceID(rdhl)).str : "N/A",
+//       nSPages, nCRUPages, nTimeFrames, nHBFrames, int(blocks.size()), nErrors);
+//  if (verbose) {
+//    for (int i = 0; i < int(blocks.size()); i++) {
+//      std::stringstream counts;
+//      counts << '#' << std::setw(5) << i << " | ";
+//      blocks[i].print(counts.str());
+//    }
+//  }
```

Comment:
@ktf why did you comment this?

Reply:
no reason just to reduce noise while debugging. I still need to clean this pr up.